init.sql

```diff
@@ -2,19 +2,20 @@ CREATE TABLE users (
     id VARCHAR(255) PRIMARY KEY,
     username VARCHAR(255) UNIQUE NOT NULL,
     password VARCHAR(255) NOT NULL,
-    created_at TIMESTAMP NOT NULL,
-    updated_at TIMESTAMP,
-    deleted_at TIMESTAMP,
+    created_at TEXT NOT NULL,
+    updated_at TEXT,
+    deleted_at TEXT,
```

Comment:
why did we change these to texts?

Reply:
Changed to texts because the time.Now().String() function returns a string, and while it is possible to define it as a timestamp object in a gorm struct, I saw no advantage of doing so and simply making it a string was easier so I took the path of least resistance.